CONTRIBUTING.md

```diff
@@ -41,313 +27,228 @@ process documented here.
 
-    This is where you actually make changes to your private copy of the
-    repository.
+1. Fork the [upstream repository](https://help.github.com/articles/cloning-a-repository):
 
-- [Setup a "git remote"](https://help.github.com/articles/configuring-a-remote-for-a-fork):
+1. [Clone your forked copy of the upstream repository](https://help.github.com/articles/cloning-a-repository):
 
-  "What happened to my fork?". The following demonstrates where your fork is
-  and how you use it.
+1. While on your *forked copy*, select the green button `Clone or download`
+   and copy the URL. 
 
-  Recall that your fork on the GitHub server is an *exact copy* of the
-  upstream repository that also lives on the GitHub server. That might not be
-  true moving forward because PRs are constantly merged into the upstream
-  repository. This means your fork will be out of date because it will not
-  contain the latest changes in the upstream repository. The same
-  problem also affects your local copy of the upstream repository.
+1. Run the commands below and **paste the copied URL** (previous step),
+   so your real GitHub user name replaces `your-github-username` below.
+  
+```sh
+$ dir="$GOPATH/src/github.com/kata-containers" 
+$ mkdir -p "$dir"
+$ cd "$dir"
+$ git clone https://github.com/{your-github-username}/community
+$ cd community
+```
+   
+**Note:** Cloning a forked repository automatically gives a remote `origin`.
 
-  The way you keep your local copy up to date is to run `git pull`
-  periodically. This command downloads all the latest changes from "another
-  repository" into your local repository.
+##### Configure the upstream remote
 
-  You might ask how Git knows which repository to pull changes from. Simply,
-  it defaults to pull from the repository in which you ran `git clone`
-  previously. In Git terminology that default repository is given two names:
+Next, add the remote `upstream`. Configuring this remote allows you to
+synchronize your forked copy, `origin`, with the `upstream`. The 
+`upstream` URL varies by repository. We use the `upstream` from the Community for this example. 
 
-  - The URL for the default upstream repository is called the `origin`.
+1. Change directory into `community`. 
 
-  - The local branch that Git puts the copy of the upstream repository into is
-    called `master`.
+1. Set the remote `upstream` as follows. 
 
-  If you run `git push`, Git will try to upload any of your local changes
-  into the default repository (i.e. the `origin`). This will not work because
-  the `origin` is the upstream repository that you do not have permission to
-  change.
+    ```sh
+    $ git remote add upstream https://github.com/kata-containers/community
+    ```
 
-  You need to tell Git that your changes should be uploaded to your fork on
-  the GitHub server when you `git push` This is done by adding what is called
-  a "remote":
+1. Run `git remote -v`. Your remotes should appear similar to these. 
```

Comment:
Nit: Could you change the final period to a colon?

Reply:
Done.

Reply:
There was one last note that needed a formatting correction; done in latest commit.